src/clj/comimo/views.clj

```diff
@@ -1,12 +1,12 @@
 (ns comimo.views
   (:require
-   [clojure.data.json   :as json]
-   [clojure.java.io     :as io]
-   [clojure.string      :as str]
-   [cognitect.transit   :as transit]
-   [comimo.git          :refer [current-version]]
-   [hiccup.page         :refer [html5 include-css include-js]]
-   [triangulum.config   :refer [get-config]]
```

Comment:
We prefer to keep this column alignment. See http://dev-docs.sig-gis.com/tutorials/sig-tutorials/clojure/coding-style-guide.gmi for more details.